apps/web/components/booking/BookingListItem.tsx

```diff
@@ -256,10 +256,9 @@ function BookingListItem(booking: BookingItemProps) {
       },
     });
   };
-  const showRecordingsButtons =
-    (booking.location === "integrations:daily" || booking?.location?.trim() === "") && isPast && isConfirmed;
 
   const title = booking.title;
+  const showRecordingsButtons = booking.isRecorded && isPast && isConfirmed;
```

Comment:
All existing bookings would stop showing recording buttons as they don't have isRecorded set.
We should have legacy bookings handling as well ideally.

Reply:
Yeah we would have to create a query that would run when we deploy the site

Reply:
2nd point here https://threads.com/34451715693?s=8gjkpjXYyHa7T3p4rGENJz